source/lambda/customResourceKendraDataSource/lambda_function.py

```diff
@@ -20,56 +20,58 @@ def on_create(event, context):
     )
     
     print("covid FAQ created")
-                                            
-    create_data_source_response = kendra_client.create_data_source(
-        Name='DUSCovidDataset',
-        IndexId= kendraIndexId,
-        Type='S3',
-        Configuration={
-            'S3Configuration': {
-                'BucketName': dataBucketName,
-                'InclusionPrefixes': [
-                    'documents/'
-                ]
+    
+    # create s3 folders
+    s3_client = boto3.client('s3', region_name=os.environ['AWS_REGION'])
+    s3_client.put_object(Bucket=os.environ['BULK_PROCESSING_BUCKET'], Key=('documentDrop' +'/'))
+    s3_client.put_object(Bucket=os.environ['BULK_PROCESSING_BUCKET'], Key=('kendraPolicyDrop' +'/'))
+    
+    # copy the covid pdfs into the bulk processing bucket for processing
+    # get a list of all covid pdfs and their associated kendra policy
+    response = s3_client.list_objects_v2(Bucket=dataBucketName, MaxKeys=1000)
+
+    s3_resource = boto3.resource('s3', region_name=os.environ['AWS_REGION'])
+
+    # copy over in the bulk bucket the kendra json policy files
+    for doc in response['Contents']:
+        
+        if doc['Key'].split('.')[-1] == 'json':
+            
+            copy_source = {
+                'Bucket': dataBucketName,
+                'Key': doc['Key']
             }
-        },
-        Description='Covid pdfs data',
-        RoleArn=roleArn
-    )
-    dataSourceId = create_data_source_response['Id']
-    print("Data source id {} for kendra index with id {} created.".format(dataSourceId, kendraIndexId))
-    print("syncing the s3 data source")
-    response = kendra_client.start_data_source_sync_job(
-        Id=dataSourceId,
-        IndexId=kendraIndexId
-    )
-    print("Started sync job for data source id :{}, the execution id of the sync job is :{}".format(dataSourceId,response['ExecutionId']))
+            
+            destinationKey = doc['Key'].split('/')[2]
+
```

Comment:
Could you please add logging (or print) statements here here : 
"Copying policy files for covid bucket"